### DISCUSSION TITLE
Enable security upgrades for major versions but default for minor versions

### DISCUSSION THREAD

**Discussion by ghsatpute-freshworks** [renovatebot/renovate#17754] [Request Help]
In our use case, we have enabled minor version upgrades. In addition, we would like Renovate to raise dependency upgrades only in the case where there is a security issue and a major upgrade is also available. 

If a security issue is not there, it should not raise the PR. And for minor versions, it should continue raising PRs even if no security issues are there. How can I achieve that?

```
  "packageRules": [
    {
      "matchUpdateTypes": ["major"],
      "enabled": true // But only for security updates
    },
    {
      "matchUpdateTypes": ["minor", "patch"],
      "matchDepTypes": ["devDependencies", "dependencies"],
      "stabilityDays": 5,
      "prCreation": "not-pending",
      "automerge": true
    }
```

**Comment by rarkins** [comment]
Instead of disabling, set dependencyDashboardApproval=true instead. That way you still gain visibility into ones you're missing plus can get security updates if necessary

**Reply by ghsatpute-freshworks** [reply]
Thanks for the reply @rarkins. 
Is there any way can differentiate the security ones from the normal ones?
<img width="793" alt="image" src="https://user-images.githubusercontent.com/82932897/189715633-b62809ec-2ae7-46ac-957c-f82b6304b169.png">

**Reply by rarkins** [reply]
1. They won't be rate limited 
2. They'll have the characteristics from here: https://github.com/renovatebot/renovate/blob/c4fa239bfaf61deef30c7d2f95afd77f5d4a7acf/lib/config/options/index.ts#L1582
